bedrock/base/templates/base.html

```diff
@@ -97,12 +97,21 @@
 
         {% block site_header_nav %}
         <nav id="nav-main" role="navigation">
-            <ul>
-                <li><a href="{{ url('mozorg.mission') }}">{{_('Mission')}}</a></li>
-                <li><a href="{{ url('mozorg.about') }}">{{_('About')}}</a></li>
-                <li><a href="{{ url('firefox.family.index') }}">{{_('Products')}}</a></li>
-                <li><a href="{{ url('mozorg.contribute') }}">{{_('Get Involved')}}</a></li>
-            </ul>
+          <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
+          <ul id="nav-main-menu">
```

Comment:
I believe this ID is superfluous as well, as it's only used by the responsive nav JS and to associate the toggle control.